notebook/static/tree/js/notebooklist.js

```diff
@@ -595,6 +597,20 @@ define([
             $('.delete-button').css('display', 'none');
         }
 
+        // View is visible when an item is renderable or downloadable
```

Comment:
Oh, I meant literally "can't" because I assumed the browser wouldn't let us open a bunch of new tabs/windows. Does it really work?